newIDE/app/src/MainFrame/Onboarding/OnboardingDialog.js

```diff
@@ -27,25 +23,45 @@ const styles = {
   },
 };
 
+const USERFLOW_DEV_ID = 'ct_y5qogyfo6zbahjejcbo3dybnta';
+const USERFLOW_PROD_ID = 'ct_paaz6o2t2bhlrlyi7a3toojn7e';
+
+const userFlowIds = {
+  fr_FR: '761ac722-d74e-45e3-a870-cd0f60f3299f',
+  pt_BR: 'a31fa85f-df42-40b5-aa80-3c6a59ad9666',
```

Comment:
Should we also use it for pt_PT?

Reply:
I'll check with Luni, but I assume so, good catch